src/main/gov/nasa/jpf/jvm/ClassFilePrinter.java

```diff
@@ -224,6 +224,22 @@ public void setRecordComponentAttribute(ClassFile cf, Object tag, int componentI
     pw.println();
   }
 
+  @Override
```

Comment:
Please try to isolate this chunk and revert all the other whitespace changes, in case we have to merge this patch later on other branches like `java-11`.